### ISSUE TITLE
Build a binary for Apple Silicon

### ISSUE THREAD

**Issue by saran-t** [deepmind/mujoco#2]
MacOS binary releases should also be available for Apple Silicon machines.

**Comment by matte-esse** [comment]
I don't want to open another duplicate but I do want to add another request for Apple Silicon binaries!

**Comment by saran-t** [comment]
We have a working (and performant) universal binary build for macOS and have repackaged our macOS release into a proper [Framework Bundle](https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPFrameworks/Concepts/FrameworkAnatomy.html), meaning that people will be able to use it more easily in Xcode projects (including Swift ones).

We are now working through our internal release process and intend to make the new build available within the next few weeks.

**Comment by bermanboris** [comment]
Thanks a lot!

**Comment by wookayin** [comment]
Excited to hear the great news of [the 2.1.1 release](a499b3812039dfbc6bc0d4cb86ce888280809abd)!  Given that MuJoCo is provided as a framework bundle, which is different from what we used to have before (i.e. `~/.mujoco/mujoco210`), would you please provide installation instructions for end users so that one can use other mujoco bindings such as mujoco-py?

**Comment by saran-t** [comment]
This issue is about mujoco-py, which isn't a project that we own or maintain. Having said that, since MuJoCo 2.1.1 doesn't change any public API compared to 2.1.0 we expect existing bindings to continue to work modulo changes in file names and location.

For dm_control, we will very shortly be pushing a small change that fixes the load path for MuJoCo and provide updated installation instructions. For mujoco-py, if the maintainers would like to update their code we'd be more than happy to work with them on it. In the meantime, I suggest that you mount the DMG for 2.1.1 then try the following:

```shell
mkdir -p ~/.mujoco/mujoco210/bin
cp /Volumes/MuJoCo/MuJoCo.framework/Versions/Current/libmujoco.2.1.1.dylib ~/.mujoco/mujoco210/bin/libmujoco210.dylib
cp -r /Volumes/MuJoCo/MuJoCo.framework/Versions/Current/Headers ~/.mujoco/mujoco210/include
```

Let us know if you get anywhere with this!

**Comment by wookayin** [comment]
@saran-t Yes I know my question is a bit off-topic. However, I think it's still worth considering having a canonical/standard installation path so that downstream applications can detect the lib properly, and in regard to that my point is mainly about *installation process* and relevant documentation.

Do you expect users to use MuJoCo where the image is being mounted (which is not permanent)? Why don't we have MuJoCo.framework *installed* somewhere into the system? We could consider providing an installer, but I think simply letting `MuJoCo.app` installed to `/Applications` would be an easier solution (if the volume provides a conventional drag-drop instruction..).

**Comment by saran-t** [comment]
This is our proposed installation scheme for `dm_control`: https://github.com/deepmind/dm_control/commit/84fc2faa95ca2b354f3274bb3f3e0d29df7fb337

Note that this is likely a somewhat temporary solution while we work out proper installation setup (we're considering msi for Windows, deb/rpm for Linux, and possibly pkg for macOS).

**Comment by ikasou** [comment]
I have submitted a pull request for openai/mujoco-py that adds support for Xcode clang and uses the new MuJoCo.framework:
[https://github.com/openai/mujoco-py/pull/664]
If anyone wants to try it locally and give some feedback,
git clone -b m1 https://github.com/ikasou/mujoco-py.git
pip3 install -U -e mujoco-py
